deploy/infra/modules/aks-invoker-role-assignment.bicep

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+param principalId string
+param aksClusterName string
+param customRoleDefId string
+param roleAssignmentId string = guid(principalId, aksClusterName, customRoleDefId)
+
+resource aks 'Microsoft.ContainerService/managedClusters@2021-10-01' existing = {
+  name: aksClusterName
```

Comment:
can we get the `principalId` from this aks resource, that ways we will be able to reduce a parameter.